app/Models/AwardCat.php

```diff
@@ -45,11 +45,11 @@ class AwardCat extends Model
 
     protected $guarded = [];
 
-    public function awardpage(){
+    public function awardpage() {
         return $this->hasOne('App\Models\AwardPage', 'id', 'award_page_id');
     }
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->